cli/flox/doc/flox-config.md

```diff
@@ -7,69 +7,111 @@ header: "flox User Manuals"
 
 # NAME
 
-flox-config - configure user parameters
+flox-config - view and set configuration options
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] config ([(-l|\--list)] | (-r|\--reset) | --set `<key>` `<value>` | --set-number `<key>` `<number>` | --set-bool `<key>` `<bool>` | --delete `<key>`)
+```
+flox [ <general-options> ] config
+     [-l |
+      -r |
+      --set <key> <string> |
+      --set-number <key> <number> |
+      --set-bool <key> <bool> |
+      --delete=<key>]
+```
 
 # DESCRIPTION
 
-Configure and/or display user-specific parameters.
+Without any flags or when `-l` is passed, `flox config` shows all options with
+their computed value.
+
+Config values are read from the following sources in order of descending priority:
+
+1. Environment variables.
+   All config options may be set by prefixing with `FLOX_` and using
+   SCREAMING_SNAKE_CASE.
```

Comment:
```suggestion
   All config options may be set by prefixing with `FLOX_`
   and using SCREAMING_SNAKE_CASE.
```

From what I understand reading the [sembr docs](https://sembr.org/#:~:text=We%20can%20further,spirit%20of%20brotherhood.) is that you _can_ break before an "and".
I usually extend that to "or" as well, just trying to find the furthest point to the 80 char line like this that I can break at while keeping most of the semantics grouped together.
I find this relatively intuitive but this whole spec leaves a lot of subjective wiggle room!

Reply:
I think it has more to do with whether the and is joining two clauses than the words "and" or "or"